### DISCUSSION TITLE
Shared dependencies

### DISCUSSION THREAD

**Discussion by taihojwn** [renovatebot/renovate#10288] [Request Help]
Hi all,
Quick question regarding how the Renovate Bot determines what version of a dependency it should bump to in the case of dependency A and dependency B rely on different versions of another dependency C.

Does Renovate only bump that dependency C to a version that satisfies both A and B?

The Renovate Bot is a great tool!

Thanks

**Comment by viceice** [comment]
By default it opens a PR for bothe to latest major and separate prs for latest minor or patch

**Comment by rarkins** [comment]
If dependency C is a transitive dependency then Renovate doesn't do any logic at all about it other than calling the relevant package manager if necessary

**Comment by taihojwn** [comment]
oh, I see. Does that mean we're responsible for making any code changes to ensure the project can still compile/work in those separate PRs before we merge? I was thinking it's kind of cool if Renovate only bumps things to the version that the no syntax changes required

**Reply by viceice** [reply]
It really depends on your setting, if you provide a more full sample we can help to configure things for you.

**Reply by rarkins** [reply]
You're better off to know that there is an upgrade available which you need to make changes after than instead having it silently dropped because it's incompatible.
